conformance/utils/roundtripper/roundtripper.go

```diff
@@ -43,6 +46,9 @@ type Request struct {
 	Method           string
 	Headers          map[string][]string
 	UnfollowRedirect bool
+	CertPem          []byte
+	KeyPem           []byte
+	Server           string
```

Comment:
can the Host field be reused ?

Reply:
I considered `Host` and `Server` to be different.  `Server` is one of the `TLSRoute.Spec.Hostnames` that aligns with `CertPem` and `KeyPem`, and is passed as the `TLSClientConfig.ServerName`.   Since `TLSRoute.Spec.Hostnames` can contain wildcards, it might not match the host.  Ie, we could have a TLS server *.example.com but a request host abc.example.com.